src/style/style.css

```diff
@@ -45,7 +45,11 @@ body {
 }
 
 .one_btn{
-    margin-top: 15%;
+/*PROBLEM: THIS POSITION ISNT WORKING, NEED TO HAVE IT STAY IN ONE PLACE*/
```

Comment:
Is this still a problem?

Reply:
Going over this again, realized we talked about this on Sunday and decided it's fine. Merging now, great work!!!!

Reply:
Going over this again, realized we talked about this on Sunday and decided it's fine. Merging now, great work!!!!